pop-api/examples/fungibles/tests.rs

```diff
@@ -808,7 +833,7 @@ mod utils {
 		}
 	}
 
-	/// Convert into `PSP22Error::Custom` error type.
+	// Convert into `PSP22Error::Custom` error type.
```

Comment:
```suggestion
	/// Convert into `PSP22Error::Custom` error type.
```
Shouldn't this is `///`?